### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by Zuwairanajma**
In this branch:-
I enhanced the styling.
Added a media query for a small screen.
and gave some final touches.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,194 @@
...
+### Key Features <a name="key-features"></a>
+- **Navbar (menu-bar)**
+- **localStorage functionality**
+- **Add book functionality**
+- **Remove book functionality**
+- **Footer section**
+- **contact information**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (Efiamotu-1) [inline README.md:61]: [OPTIONAL]
- [ ] Your project currently doesn't have a live demo and as such you either provide one or you remove this section. When removing this section also take note to remove the section from the headers at the  top of your readme file. this change is important so as to make your readme very professional and not have sections which are redundant :+1:

File: src/index.css
Comment (Efiamotu-1) [inline src/index.css:1]: - [ ] Your project currently doesn't match completely the provided wireframe as shown below. Your design is expected to cover the entire view of the users screen but it is only occupying a small section as shown below. I kindly suggest you increase the width to achieve the project design 👍 ✨ 
- > see this [project link](https://efiamotu-1.github.io/Leaderboard/dist/) as example
- project wireframe
wfi
- your design
cv

File: README.md [hunk 2]
```diff
@@ -1 +1,194 @@
...
+<details>
+ <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/JS">JS</a></li>
+    <li><a href="#">WebpackDevServer</a></li>
+    <li><a href="#">HTMLWebpackPlugin</a></li>
+  </ul>
+</details>
```
Comment (Efiamotu-1) [inline README.md]: [OPTIONAL] 
- [ ] Some Technologies listed here have already been mentioned above. I kindly suggest you remove duplicated technologies from this section

File: README.md [hunk 3]
```diff
@@ -1 +1,194 @@
...
+
+I would like to appreciate and acknowledge the esteem Microverse program for providing me with necessary template files and detailed instructions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+**No FAQ for now**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Efiamotu-1) [inline README.md]: - [ ] I kindly suggest that since you do not have an FAQ for now you can remove this optional section 👍 ✨

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Zuwairanajma , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @Zuwairanajma, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**